orchestra/models/core/models.py

```diff
@@ -659,9 +660,21 @@ class Todo(TodoMixin, BaseModel):
     class Meta:
         app_label = 'orchestra'
 
-    task = models.ForeignKey(
-        Task, related_name='todos', on_delete=models.CASCADE)
+    class Status(ChoicesEnum):
+        PENDING = 'pending'
+        COMPLETED = 'completed'
+        DECLINED = 'declined'
+
+    task = models.ForeignKey(Task, null=True, blank=True,
+                             related_name='todos', on_delete=models.CASCADE)
```

Comment:
```suggestion
                             related_name='todos', on_delete=models.SET_NULL)
```

Reply:
Now that task is less prominent, we should make it so deleting a task doesn't delete a todo